File: app/src/main/java/ru/vpiska/app/ApiClient.java
package ru.vpiska.app;

import okhttp3.OkHttpClient;
import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class ApiClient {

    private static Retrofit retrofit = null;
    public static int unique_id;
    private final static String URL_SERVER = "https://clickcoffee.ru/api/v2/";

    public static Retrofit getClient() {

        OkHttpClient okHttpClient = UnsafeOkHttpClient.getUnsafeOkHttpClient();
        if (retrofit==null) {
            retrofit = new Retrofit.Builder()
                    .baseUrl(URL_SERVER)
                    .client(okHttpClient)
                    .addConverterFactory(GsonConverterFactory.create())
                    .build();
        }
        return retrofit;
    }
}

File: app/src/main/java/ru/vpiska/app/ApiClient.kt
package ru.vpiska.app

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object ApiClient {
    private const val URL_SERVER = "https://clickcoffee.ru/api/v2/"

    val client: Retrofit by lazy {
        Retrofit.Builder()
                .baseUrl(URL_SERVER)
                .client(UnsafeOkHttpClient.unsafeOkHttpClient)
                .addConverterFactory(GsonConverterFactory.create())
                .build()

    }
}